build_tools/s3.py

```diff
@@ -77,21 +77,30 @@ def get_files(archive_path, bucket, sha1):
     files = files + find_files_in_bucket(archive_path, bucket, sha1, "editor-alpha", '.*(/Defold-.*)$')
     return files
 
-def get_tagged_releases(archive_path):
+def get_tagged_releases(archive_path, pattern=None):
     u = urlparse.urlparse(archive_path)
     bucket = get_bucket(u.hostname)
 
+    if pattern is None:
+        pattern = "(.*?)$" # captures all tags
+
     tags = run.shell_command("git for-each-ref --sort=taggerdate --format '%(*objectname) %(refname)' refs/tags").split('\n')
     tags.reverse()
     releases = []
+
+    matches = []
     for line in tags:
         line = line.strip()
         if not line:
             continue
-        m = re.match('(.*?) refs/tags/(.*?)$', line)
+        p = '(.*?) refs/tags/%s' % pattern
+        m = re.match('(.*?) refs/tags/%s' % pattern, line)
         if not m:
             continue
         sha1, tag = m.groups()
+        matches.append((sha1, tag))
+
+    for sha1, tag in matches[:10]: # Only the first releases
```

Comment:
Some gymnastics to not go through nearly 200 releases, since it takes a few minutes, and we only wish to release the last ~10 anyways.